payu/experiment.py

```diff
@@ -721,8 +703,8 @@ def run(self, *user_flags):
             error_log_dir = os.path.join(self.archive_path, 'error_logs')
             mkdir_p(error_log_dir)
 
-            # NOTE: This is only implemented for PBS scheduler
-            job_id = scheduler.get_job_id(short=False)
+            # NOTE: This is PBS-specific
+            job_id = self.scheduler.get_job_id(short=False)
```

Comment:
I think this should be accessible in the telemetry data stored in `set_run_info` above.